pom.xml

```diff
@@ -154,6 +154,20 @@
       <groupId>com.google.cloud.bigdataoss</groupId>
       <artifactId>bigquery-connector</artifactId>
       <version>${bigquery.connector.hadoop2.version}</version>
+      <exclusions>
```

Comment:
can you add a comment about why these need to be excluded

Reply:
Done.